applications/nrf_desktop/README.rst

```diff
@@ -339,7 +339,9 @@ The project comes with configuration files for the following boards:
     * PCA20037 - nRF Desktop keyboard reference design
     * PCA20044 - nRF Desktop casual mouse reference design (nRF52832)
     * PCA20045 - nRF Desktop casual mouse reference design (nRF52810)
-    * PCA10059 - nRF Desktop dongle reference design
+    * PCA10059 - nRF Desktop dongle reference design (nRF52840)
+    * PCA10111 - nRF Desktop dongle reference design (nRF52833)
+    * PCA10100 - nRF Desktop dongle reference design (nRF52833 DK)
```

Comment:
PCA10100 is the nRF52833 DK, not a reference design.

Reply:
On the right is what application does with it. In case of the PCA10111 it is HID dongle.
We don't try to explain what is the name of the board or the original purpose but tell what role application takes when loaded onto it.